src/components/events/Events.js

```diff
@@ -200,30 +228,89 @@ const Events = () => {
           onButtonClick={loadGetEvents}
           onResetSearch={handleRefreshSearch}
         />
+        <br />
+        <Typography
+          style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+        >
+          Filters
+        </Typography>
+        <DropdownFilter
+          placeholder="Select the Status"
+          value={status}
+          onChange={handleStatus}
+          lists={statusList}
+        />
+        <br />
+        <NumberFilter
+          placeholder="Select the Maximum Number of Participants"
+          value={filterParticipants}
+          onChange={handleFilterParticipants}
```

Comment:
Whats the use case for this filter? I'm not sure users will want to search for events based on an exact number of max participants.

What are your thoughts on altering it so that you are searching for events that still have space in the participants list? Ie Are not full

Reply:
my thought process is that the user is able to remove events from the results of the search that are too large (for example, the user is only looking for events that have 50 or less participants)

Reply:
yeah it is looking for the exact MAX participants but i can try doing the other way!

Reply:
you're right, it should be looking for less than the input, not exactly the input. @Eshah7

Reply:
Updated the feature!